src/app/_blocks/EpisodeContent/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { PodcastEpisode } from "@/payload/payload-types";
+
+export const EpisodeContent: React.FC<{ episode: PodcastEpisode }> = ({ episode }) => {
```

Comment:
Just a heads up that the React.FC type isnt totally necessary. You can just do 
``` 
const EpisodeContent = ({ episode }: { episode: PodcastEpisode }) => ...
```
But it's up to you 👍